integration/bridge.go

```diff
@@ -39,7 +39,7 @@ type bridge struct {
 
 func newBridge(addr string) (*bridge, error) {
 	b := &bridge{
-		inaddr:  addr + ".bridge",
+		inaddr:  addr + "0",
```

Comment:
Can this do something with the address like `"bridge-" + addr`? The 0 seems like it could conflict with other addresses (e.g., c+basePort = 50, pid=50 => ":100"; c+basePort=5, pid=5, bridge => ":100") . Or will that cause cert name conflicts?